### ISSUE TITLE
rename/refactor symbol?

### ISSUE THREAD

**Issue by doofin** [idris-hackers/idris-mode#481]
Sorry if i miss some parts of the readme,but i can not found how to  do rename/refactor

**Comment by jsoo1** [comment]
I do not think this is currently supported. Most of the semantic information we get comes from the compiler.  This would certainly be nice to have in the future.

**Comment by david-christiansen** [comment]
It could be done pretty easily in the scope of a single file, by finding names with the appropriate overlay property/highlighting, but that kind of misses the point of this kind of feature, which is that it works on a project scope. But the feature really belongs in the compiler, not the editor.

**Comment by jsoo1** [comment]
True. I suppose we keep an eye on this feature in the future.
